doc/release-notes/v2.0.3.rst

```diff
@@ -0,0 +1,219 @@
+op-build v2.0.2 as the current stable release in the 2.0.x series.
+
+It is recommended that v2.0.3 be used over any previous v2.0.x version
+due to the bug fixes contained within.
+
+
+
+Updated Packages
+----------------
+
++----------+----------+----------+-------------------------------------------+
+| Package  | Old      | New      | Platforms                                 |
+|          | Version  | Version  |                                           |
++==========+==========+==========+===========================================+
+| hcode    | hw051718 | hw052418 | zaius, p9dsu, witherspoon, romulus        |
+|          | a.op920  | a.op920  |                                           |
++----------+----------+----------+-------------------------------------------+
+| hostboot | a4eb7f02 | acedcb2b | zaius, p9dsu, witherspoon, romulus        |
++----------+----------+----------+-------------------------------------------+
+| linux    | 4.16.8   | 4.16.13  | openpower_mambo, firestone, firenze,      |
+|          |          |          | garrison, zaius, p9dsu, palmetto,         |
+|          |          |          | pseries, vesnin, witherspoon, habanero,   |
+|          |          |          | openpower_p9_mambo, zz, barreleye,        |
+|          |          |          | romulus                                   |
++----------+----------+----------+-------------------------------------------+
+| linux-   | 4.16.8   | 4.16.13  | openpower_mambo, firestone, firenze,      |
+| headers  |          |          | garrison, zaius, p9dsu, palmetto,         |
+|          |          |          | pseries, vesnin, witherspoon, habanero,   |
+|          |          |          | openpower_p9_mambo, zz, barreleye,        |
+|          |          |          | romulus                                   |
++----------+----------+----------+-------------------------------------------+
+| occ      | 77bb5e60 | 90208bbb | zaius, p9dsu, witherspoon, zz, romulus    |
++----------+----------+----------+-------------------------------------------+
+| sbe      | 8e0105e5 | a5969753 | zaius, p9dsu, witherspoon, romulus        |
++----------+----------+----------+-------------------------------------------+
+| skiboot  | v6.0.3   | v6.0.4   | openpower_mambo, firestone, firenze,      |
+|          |          |          | garrison, zaius, p9dsu, palmetto,         |
+|          |          |          | pseries, vesnin, witherspoon, habanero,   |
+|          |          |          | openpower_p9_mambo, zz, barreleye,        |
+|          |          |          | romulus                                   |
++----------+----------+----------+-------------------------------------------+
+
+Package: hcode
+--------------
+
+`Repository <https://github.com/open-power/hcode>`__
+
+Patches
+~~~~~~~
+
+Commits
+~~~~~~~
+
+hostboot (1):
+
+-  `e7a16c0c8685 <https://github.com/open-power/hcode/commit/e7a16c0c8685>`__
+   Release tag information updated for hw052418a.op920
```

Comment:
This concerns me...  You better not be putting op920 changes into op910, because it is not compatible.

Reply:
I think it is ok because the hostboot/Config.in file has the commit levels for all applicable levels and the level for op910 points to a valid hostboot/release-op910 commit level.

Reply:
@dcrowell77 don't worry, there's no OP920 code from Hostboot/OCC/SBE etc in here. It's just the buildroot and opal parts from what's in OP920.

In the merge commit there's the (only casually) terrible hack going on to force everything to be OP910 trees.